Widgets_CDAT/textStyleEditorWidget/TextStyleEditor.py

```diff
@@ -0,0 +1,253 @@
+import vcs
+import text_style_preview
+from PySide import QtCore, QtGui
+
+
+class TextStyleEditor(QtGui.QWidget):
+
+    savePressed = QtCore.Signal(str)
+    
+    def __init__(self):
+        super(TextStyleEditor, self).__init__()
+        self.textObject = None
+
+        wrap = QtGui.QVBoxLayout()
+
+        # Set up vertical align
+        self.VAGroup = QtGui.QButtonGroup()
+
+        T_button = QtGui.QPushButton()
+        T_button.setText("Top")
```

Comment:
Should probably just break these up into loops:

```
for alignment in ["Top", "Mid", "Bot"]:
    button = QtGui.QPushButton(alignment)
    button.setCheckable(True)
    vertical_align_group.add(button)
    vertical_align_layout.addWidget(button)
```

and do the same for the horizonal alignment buttons